docs/content_schemas/changing-an-existing-content-schema.md

```diff
@@ -1,50 +1,64 @@
 # Changing an existing content schema
 
-## General workflow
+## When `default_format.jsonnet` does not need updating
```

Comment:
I wonder whether "When not adding/removing top-level fields" would be a better description?